bootstrap/templates/new/rel/vm.args

```diff
@@ -1,6 +1,14 @@
 ## Add custom options here
 
+## Distributed Erlang Options
+##  The cookie needs to be configured prior to vm boot for
+##  for read only filesystem.
+
+# -name <%= app_name %>@0.0.0.0
+-setcookie <%= cookie %>
+
 ## Start the Elixir shell
+
 -noshell
 -user Elixir.IEx.CLI
 -extra --no-halt
```

Comment:
In order to fully satisfy #86, I think we should move the comment above the `-extra` option to make it even more obvious that the ordering of the options here is significant, and maybe re-word it like this:

```elixir
## Options added after -extra are interpreted as plain arguments and
## can be retrieved using :init.get_plain_arguments().
-extra --no-halt
```